docs/developer-docs/latest/plugins/graphql.md

```diff
@@ -33,36 +33,31 @@ yarn strapi install graphql
 
 </code-group>
 
-Then, start your app and open your browser at [http://localhost:1337/graphql](http://localhost:1337/graphql). You should see the interface (**GraphQL Playground**) that will help you to write GraphQL query to explore your data.
+Then, start your app and open your browser at [http://localhost:1337/graphql](http://localhost:1337/graphql). You should now be able to access the **GraphQL Playground** that will help you to write your GraphQL queries and mutations.
 
-## Configurations
+## Configuration
 
-By default, the [Shadow CRUD](#shadow-crud) feature is enabled and the GraphQL path is set to `/graphql`. The Playground is enabled by default for both the development and staging environments, however it is disabled in production. By changing the config option `playgroundAlways` to true, you can enable it.
+| param              | description                                                                                                                                                   | type    | default |
+| ------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------- | ------- |
```

Comment:
Thanks for the table @gu-stav. Headers mention 4 columns but the body only uses 3 :-) It would be better to split the description and type (description first, then type, just like in the headers).

Also, titles for the table should be title cased, and no abbreviation should be used ("param" → "parameter").

Reply:
Fixed in f7250f2097ec6d114703cf99fb0b73294c7b4d0b